dune-project

```diff
@@ -1,2 +1,12 @@
-(lang dune 1.9)
+(lang dune 1.10)
+(name citty)
 (using fmt 1.2)
+
+(generate_opam_files true)
+(source (github ocaml-ci/citty))
+(authors "Frédéric Bour")
+
+(package
+ (name citty)
+ (synopsis "CI in TTY")
+ (depends lwt notty fpath mirage-flow-unix))
```

Comment:
Why does it depend on `mirage-flow-unix`?

Reply:
I copied over these dependencies from the README: https://github.com/ocaml-ci/citty#2-install-dependencies. I think it's not necessary to depend directly on `capnp-rpc-unix`, since this repository contains it as a submodule?

Reply:
Well, it won't contain it if someone updates to the latest version of ocaml-ci, and it won't work if someone tries to publish citty to opam.

Reply:
Good points; I'll fix the .opam file and the README accordingly